src/openrct2/world/Location.hpp

```diff
@@ -702,6 +702,44 @@ struct TileCoordsXYZD : public TileCoordsXYZ
     }
 };
 
+struct BoundBoxXY
```

Comment:
Don't put these in location.hpp they are only used by paint functions. Suggest making a new header just for them as no doubt more things will be added to them along the way.

Reply:
What do you think the best location for this new header might be? `world` would probably work, but I'm not sure if you might have another better location in mind.

Reply:
Stick it under paint since it's paint related